tests/includes/models/test-class-accredible-learndash-model-auto-issuance-log.php

```diff
@@ -65,6 +65,19 @@ public function test_get_results() {
 		$results = Accredible_Learndash_Model_Auto_Issuance_Log::get_results( '', $limit, $offset );
 		$this->assertCount( 1, $results );
 		$this->assertEquals( $data2_id, $results[0]->id );
+
+		// With $limit & $offset.
```

Comment:
Duplicate test case.

Reply:
Thanks for the catch! I removed the duplicate test case.